scrapers/PaperStreetMedia.yml

```diff
@@ -31,5 +34,12 @@ xPathScrapers:
       Performers:
         Name: //p[@class="model-name"]/a
       Image: //div[@class="video-area"]//stream/@poster
-
-# Last Updated July 21, 2020
+      Studio:
+        Name:
+          selector: //li[@class="nav-item login"]/a[@class="nav-link px-0"]/@href
```

Comment:
the login selector is not exact enough
for `teensloveanal, exxxtrasmall, dyked, stepsiblings, innocenthigh, badmilfs` the login points to **teamskeet** and thus the studio becomes **teamskeet**.

we can use instead 
- the canonical link
```yaml
        Name:
          selector: //link[@rel="canonical"]/@href
          postProcess:
              - replace: 
                   - regex: (.+www\.)(\w+)(.+)
                    with: $2
```
- or the join link
```yaml
        Name:
          selector: //p[@class="video-description"]/a[@class="join-link"]/@href
          postProcess:
              - replace: 
                  - regex: (.+join\.)(\w+)(.+)
                    with: $2
```